File: app/src/main/java/io/vextil/launcher/Application.java
package io.vextil.launcher;

import io.paperdb.Paper;
import io.vextil.launcher.managers.AppsManager;
import io.vextil.launcher.managers.WebAppsManager;

public class Application extends android.app.Application {

    public static AppsManager appsManager;
    public static WebAppsManager webAppsManager;

    @Override
    public void onCreate() {
        super.onCreate();
        Paper.init(this);
        appsManager = new AppsManager(getApplicationContext());
        webAppsManager = new WebAppsManager(getApplicationContext());
    }

}

File: app/src/main/java/io/vextil/launcher/Application.kt
package io.vextil.launcher

import android.app.Application
import io.paperdb.Paper
import io.vextil.launcher.managers.WebAppManager
import io.vextil.launcher.models.App

class Application : Application() {

    override fun onCreate() {
        super.onCreate()
        Paper.init(this)
    }

}